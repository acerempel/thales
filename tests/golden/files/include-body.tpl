<h1>Oculi Omnium</h1>
<p>
{let template = "include-body/body.tpl" in}{include-body load-template(template)}{end}
</p>
