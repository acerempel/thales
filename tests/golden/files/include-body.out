<h1>Oculi Omnium</h1>
<p>
Oculi omnium ad te sperant, Domine. Tu das escam illorum
in tempore opportuno. Oportes manum tuum et imples quicquid
vivit benedictionem tuam.

</p>
